lib/bummr/outdated.rb

```diff
@@ -1,17 +1,19 @@
 require 'open3'
-require 'singleton'
 
 module Bummr
   class Outdated
-    include Singleton
+    def initialize(file_reader: File, process_provider: Open3)
```

Comment:
By using default arguments here the production code can simply call `Bummr::Outdated.new` without any arguments, but in our tests we can pass in instance / class doubles as appropriate.